gc/base/GCExtensionsBase.hpp

```diff
@@ -366,6 +369,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	float heapSizeStartupHintConservativeFactor; /**< Use only a fraction of hints stored in SC */
 	float heapSizeStartupHintWeightNewValue;		/**< Learn slowly by historic averaging of stored hints */	
 
+
```

Comment:
can you remove this extra blank line?

Reply:
NVM about this unless @amicic has other changes that he would like to see added.